app/presenters/green_lanes/assessments_presenter.rb

```diff
@@ -26,6 +26,14 @@ def cat_2_exemptions
       extract_exemptions(cat2_with_exemptions)
     end
 
+    def cat_1_assessments
```

Comment:
Probably some refactoring can happen with this file to be honest.. as there are some bits which look repeated but rather get things working before that happens.

Reply:
do we need for `cat1_with_exemptions`
and `cat1_with_exemptions` ? :thinking: 
also, where are they used?